.github/workflows/test.yml

```diff
@@ -10,6 +10,9 @@ jobs:
       matrix:
         node-version: [14.x]
 
+    # We want to run on external PRs, but not on our own internal PRs as they'll be run
+    # by the push to the branch.
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
Not sure if I understand this. Won't our own PRs trigger if we just set the `on` to:

```yml
on:
  pull_request:
    branches:
      - master
```

Reply:
Can you clarify what you mean?  I don't think we want our PR's to trigger to this, only people who are making PR's from other repos.